### ISSUE TITLE
Create Table able to have Two Columns with Same Name

### ISSUE THREAD

**Issue by saklanipankaj** [aprismatic/prismadb#4]
```
CREATE TABLE t2
(
    **a INT ENCRYPTED FOR (MULTIPLICATION, ADDITION, RANGE),**
    b FLOAT8 ENCRYPTED FOR (MULTIPLICATION, ADDITION, RANGE),
    **a INT,**
    d VARCHAR(10) ENCRYPTED FOR (STORE, SEARCH),
    e VARCHAR(30)
);
```
A table is able to be created with two columns with the same name, when one column is an encrypted column. This should not be allowed, an error should be thrown for this SQL Statement.

**Comment by cheziyi** [comment]
What is the resulting schema after the table is created?

**Comment by saklanipankaj** [comment]
Field |    Type     | Null | Key | Default | Extra |           Encryption
-------+-------------+------+-----+---------+-------+---------------------------------
 a     | int4        | YES  |     |         |       | None
 b     | float8      | YES  |     |         |       | Addition, Multiplication, Range
 d     | varchar(10) | YES  |     |         |       | Store, Search
 e     | varchar(30) | YES  |     |         |       | None
(4 rows)

**Comment by cheziyi** [comment]
Noted, will fix.

**Comment by cheziyi** [comment]
Fixed in MySQL 0.3.3.75, SQL Server 0.3.3.102, PostgreSQL 0.3.3.37
